src/main/java/seedu/duke/command/ListCommand.java

```diff
@@ -1,17 +1,38 @@
 package seedu.duke.command;
 
+import seedu.duke.model.Item;
 import seedu.duke.model.ItemContainer;
 
 public class ListCommand extends Command {
 
+    private static final String ITEM_INFO = "%o. %s (purchase cost: %s, selling price: %s)\n";
+
+    /**
+     * Retrieves list of items from ItemContainer.
+     *
+     * @param list the ItemContainer in which lsit is retrieved
+     * @return list of items
+     */
+    private String getList(ItemContainer list) {
+        String info = "";
+        for (int i = 0; i < list.getSize();  i++) {
+            Item selectedItem = list.getItem(i);
+            int index = i + 1;
+            info += String.format(ITEM_INFO, index,
+                    selectedItem.getName(),selectedItem.getPurchaseCost(),selectedItem.getSellingPrice());
```

Comment:
Should there be space between "," and "selectedItem"? Same goes to the following arguments